Cargo.toml

```diff
@@ -73,6 +77,10 @@ all = ["electrum", "esplora", "fs", "cli", "clap", "log"]
 cli = ["base64", "env_logger", "clap", "shellexpand", "fs", "serde", "electrum", "esplora", "log"]
 log = ["env_logger"]
 electrum = ["bp-electrum", "serde", "serde_json"]
-esplora = ["bp-esplora"]
+# The transaction data structure required by mempool,
```

Comment:
in this case instead of putting mempool to esplora feature you need to do otherwise and add explore to mempool feature

Reply:
> in this case instead of putting mempool to esplora feature you need to do otherwise and add explore to mempool feature

After modifying according to the above suggestions, I adjust 'features' simultaneously